manifests/resource/location.pp

```diff
@@ -123,6 +122,14 @@
 #    server   => 'test2.local',
 #  }
 #
+#  Use one location in multiple serrvers
```

Comment:
Spelling of `servers`

Reply:
> Sorry about the confusion, that was basically a note for myself :D

Not for that, thank you for the review ;)

Reply:
Thx for the finding, I've fixed that.